docs/style.css

```diff
@@ -22,17 +47,20 @@
     font-display: swap;
 }
 
+/* Style */
+
```

Comment:
Nitpick: dat is wat mij betreft de extra bytes en schermruimte niet waard, uit de backets/opmaak/code zelf is dit al wel duidelijk?

```suggestion
```